<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>923</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CA Light</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="MainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>871</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QPushButton" name="creatCaCertBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сертификат УЦ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openWorkDirBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Инициализация (в текущей директории)</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="debugLogEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="certLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сертификаты:</string>
     </property>
    </widget>
    <widget class="QLabel" name="debugLogLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>251</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Дебаговое окно:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="creatCsrBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сгенерировать CSR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signCsrBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Подписать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="debugCheckContainersBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Контейнеры</string>
     </property>
    </widget>
    <widget class="QTableView" name="certTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>711</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="revokeCertBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отозвать сертификат</string>
     </property>
    </widget>
    <widget class="QPushButton" name="creatCrlBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>310</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сгенерировать CRL</string>
     </property>
    </widget>
    <widget class="QLabel" name="caLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CA: no certificate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="importBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Импорт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exportBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Экспорт</string>
     </property>
    </widget>
    <widget class="QLabel" name="dirLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="editConfigBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Файл конфигурации</string>
     </property>
    </widget>
    <widget class="QPushButton" name="installOpensslBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Скачать openssl-1.1.0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
